### PR TITLE
[DOCS] Minor edits to to tutorial for clarity

### PR DESCRIPTION
**Pull request by bobfuru**
Clarifies commands that the user needs to run in the step by step tutorial.

### CODE REVIEW

File: website/docs.md [hunk 1]
```diff
@@ -346,11 +346,13 @@ Listener names must be valid [Go identifiers][identifiers]. For example, the
...
 `"foo-bar"` are illegal.
 
-Run `go mod tidy` and then `SERVICEWEAVER_CONFIG=weaver.toml go run .`.
+Run `weaver generate`, then `go mod tidy`, and then
+`SERVICEWEAVER_CONFIG=weaver.toml go run .`.
 The program should print out the name of the application and a unique
 deployment id. It should then block serving HTTP requests on `localhost:12345`.
 
 ```console
+$ weaver generate
```
Comment (bobfuru) [inline website/docs.md:355]: I had to run `weaver generate` again before I could run the next commands, so I'm suggesting that this gets called out in the docs.

**Review by mwhittaker — APPROVED** [review]
Awesome! Thanks so much :)

File: website/docs.md (hunk 1 above)
Comment (mwhittaker) [reply website/docs.md:355]: Yup, good spot. I think previously it wasn't needed, but because of some recent changes, it is now needed.

**Comment by bobfuru** [comment]
Thanks, @mwhittaker!

**Pull request merged**